src/lib/environment.ts

```diff
@@ -113,7 +113,7 @@ export class Environment {
   @IsNotEmpty()
   public readonly PUBLIC_WEBEX_DEV_PORTAL_URL: string;
 
-  @IsUrl()
+  @IsString()
   public readonly PUBLIC_TUNNEL: string;
```

Comment:
The logic to handle the current origin of the running app can be removed. The simplest way to tell SvelteKit where the app is being served is to set the `ORIGIN` environment variable ([link](https://kit.svelte.dev/docs/adapter-node#environment-variables-origin-protocolheader-hostheader-and-port-header)).